library/src/main/scala/scala/scalajs/runtime/package.scala

```diff
@@ -97,7 +98,11 @@ package object runtime {
    *
    *  See [[LinkingInfo]] for details.
    */
-  def linkingInfo: LinkingInfo = throw new Error("stub")
+  @nowarn @deprecated("Use scala.scalajs.LinkingInfo or scala.scalajs.js.special.fileLevelThis instead", "1,17,0")
```

Comment:
Is `@nowarn` necessary? If yes, why?

Also `1.17.0` with dots:
```suggestion
  @deprecated("Use scala.scalajs.LinkingInfo or scala.scalajs.js.special.fileLevelThis instead",
      since = "1.17.0")
```

Reply:
Oh, I added it during development, and forgot to remove it, thanks!